R-package/R/lgb.train.R

```diff
@@ -82,10 +82,10 @@ lgb.train <- function(params = list(),
 
   # Setup temporary variables
   additional_params <- list(...)
-  params <- append(params, additional_params)
+  params <- append(params = params, additional_params = additional_params)
```

Comment:
these are not the keyword arguments to `append()`. You can find the arguments by running `?append` in an R session.

Please revert this change. You can leave `append()` calls unchanged, since they are R built-ins.

```suggestion
  params <- append(params, additional_params)
```